iFixit/react-commerce
# Use nextjs image component

This PR closes #106 and #111

## Changelog

- Refactors code to use Next Image component
- Adds favicon

## QA

1. Visit the Vercel PR preview
2. Visit the parts product list page
3. Verify that favicon is shown
4. Verify that images are lazy loaded
